docs/developer-docs/agents/javascript-intro.md

```diff
@@ -21,7 +21,7 @@ If you are looking for an explanation of what an agent does, see [Agent Overview
 
 ____
 
-The Internet Computer blockchain is accessible via an API that is available at [https://icp-api.io/api/v2](https://icp-api.io/api/v2). Smart contracts are able to define their own API's using the Candid Interface Declaration Language (IDL), and they will respond to calls through the public API.
+The Internet Computer blockchain is accessible via an API that is available at [https://icp-api.io/api/v2](https://icp-api.io/api/v2). Smart contracts are able to define their own API's using the Candid Interface Declaration Language (IDL), and they will respond to calls through the public API. The JavaScript agent will abstract this api for you, but you can always check the unerlying interface [here](https://internetcomputer.org/docs/current/references/ic-interface-spec/#http-call-overview).
```

Comment:
```suggestion
The Internet Computer blockchain is accessible via an API that is available at [https://icp-api.io/api/v2](https://icp-api.io/api/v2). Smart contracts are able to define their own API's using the Candid Interface Declaration Language (IDL), and they will respond to calls through the public API. The JavaScript agent will abstract this api for you, but you can always check the unerlying interface [here](../../references/ic-interface-spec/#http-call-overview).
```
We prefer relative links since those get checked at compile time for 404s